yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/redundant-indexes.psql

```diff
@@ -176,13 +176,12 @@ JOIN indexes existing
  AND redundant.indnkeyatts <= existing.indnkeyatts
 
  /*
-  * Partial index logic: consider a partial index overshadowed if there's a 
-  * full (no predicate) index on the same columns, or if the partial 
-  * predicates match exactly.
+  * Partial index logic: not consider any partial indexes in the check for
+  * redundant with existing.
   */
  AND (
     existing.partial_pred IS NULL
-    OR redundant.partial_pred = existing.partial_pred
+    AND redundant.partial_pred IS NULL
```

Comment:
What about the case if the same partial index is present twice just with different index name?

Reply:
We don't want to report any partial indexes, so not handling any cases right now. Because to check the partial_predicate of index it is complicated to understand and properly confirm if the predicate matches or not